release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -64,10 +64,10 @@ This release adds improvements related to the following components and concepts.
 ==== {op-system} PXE and ISO now live environment
 
 The PXE media and ISO available for {op-system} are now a fully live
-environment. Unlike the previous dedicated PXE media and ISO used for
-installation, the {product-title} live environment can be configured with
-Ignition and contains all the same packages as the main {op-system}, such as
-podman. This allows arbitrary scripting of pre- or post-installation workflows. For
+environment. Unlike the previous dedicated PXE media and ISO used for {op-system}
+installation, the {op-system} live environment can be configured with
```

Comment:
```suggestion
installation for {product-title} clusters on user-provisioned infrastructure, the {op-system} live environment can be configured with
```

I think it might be worth mentioning why you'd need the RHCOS media.

Reply:
@miabbott Kathryn makes a good point here. Do you have any suggestions for how to best highlight using RHCOS media?